.github/workflows/README.md

```diff
@@ -0,0 +1,27 @@
+# Workflow Management
+
+Here we have the start of a system to automatically generated workflows (currently only for transforms). 
+In general, the design is to use templates and `make` to generate/update the workflows.
+
+```
+make
+git commit -a -s -m "update workflows"
+git push
+```
+
+should be sufficient.
+
+## Transforms 
+We define a unique test workflow for each transform, based on a common template [test-transform.template](test-transform.template).
+The Makefile is used to (re)generate all workflows a necessary.  By design, workflows for a given transform should run when
+
+* anything of substance effecting operation is modified in the transform's directory tree.
+* anything in the core libraries in this repo (e.g., data-processing/lib) assuming the transform depends on these.
+* Help! the workflow should NOT run when documentation (e.g., !**.md) is changed, however this case does not seem to be working atm.
+
+When a new transform is added to the repository, 
+
+1. add it to the corresponding macro in the Makefile (in alphabetical order please). 
+1. Run `make` to create the new test .yml for the transform
```

Comment:
can we add an automatic validation that it was done?

Reply:
i gave this some thought, but I think there will need to be away to disable a transform's test, if only temporarily.  In the past this was done by renaming Makefile to something else so that the sub-directory would not be recursed into from the parent directory's Makefile.  We could do the same in the `test-transform.template` or just require that the Makefile is edited to include/uninclude a transform.  Hmm, but maybe maintaining this mode of renaming the Makefile is best to keep as it is then isolated to the transform.  If we do that, then we can do the automation you suggested above.

Reply:
...but maybe I misunderstood `automatic validation`?

Reply:
ah, ok will at a test yaml to make sure there are tests.ymls for all transforms in the repo.

Reply:
done. in test-misc.yml